project_setup/skeletonize-existing-repository.md

```diff
@@ -63,10 +66,10 @@ up the prerequisites, this involves:
 pre-commit install
 ```
 
-## Fix merge conflicts ##
+## Fix all the inevitable merge conflicts ##
```

Comment:
Somehow, this comment didn't get recorded when I put in my review.  I think this change should be reverted to the original text and the link change above that references this heading should also be reverted.

Reply:
FIxed in commit ae5f047.

Reply:
Hi Dave @dav3r, did we resolve the conversation? Also, I'm not sure if I have permissions to merge the PR?

Reply:
Yes, thank you for making the change in https://github.com/cisagov/development-guide/commit/ae5f047ebfc0ea7687ffecd1ee132cecc0d951df.  If you don't have permissions to merge this PR, one of the members of our dev team can take care of it.